api/net/Areas/Subscriber/Controllers/ReportController.cs

```diff
@@ -354,6 +355,28 @@ public async Task<IActionResult> Generate(int id, [FromQuery] bool regenerate =
         return new JsonResult(new ReportModel(report, _serializerOptions));
     }
 
+    /// <summary>
+    /// Generate an Excel document report.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpPost("{id}/export")]
+    [Produces("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [SwaggerOperation(Tags = new[] { "Report" })]
+    public FileResult Generate(int id)
```

Comment:
This method will causing confusion due to naming and location.  It doesn't generate a report, it generates an Excel document.  We have other methods named Generate which generate the report.  Also the `id` is not for a report, instead it is for an instance of a report.  Move this whole method to the `ReportInstanceController` as it's generating an Excel document based on the report instance.

- Rename to `GenerateExcel`
- Move to `ReportInstanceController`